app/src/main/res/values/colors.xml

```diff
@@ -3,4 +3,5 @@
     <color name="colorPrimary">#3F51B5</color>
     <color name="colorPrimaryDark">#303F9F</color>
     <color name="colorAccent">#FF4081</color>
+    <color name="FAB_color">#353892</color>
```

Comment:
resource keys must be in lowercase and with the word separated by _

Reply:
done